### DISCUSSION TITLE
Data Importer exit code 1 for duplicated transactions breaks my cronjob monitoring

### DISCUSSION THREAD

**Discussion by danisabin** [firefly-iii/firefly-iii#9471] [Firefly III] [answered]
Hi all!

I have setup a cronjob to automatically run Data Importer everyday, like this:

```
docker exec firefly_iii_importer php artisan importer:auto-import /import
```

I monitor that cronjob using an external service. I just send the exit code from that command, and the service notifies me if it's different from 0.
Since the latest release, I believe in this [commit](https://github.com/firefly-iii/data-importer/commit/a23c35a2b0223d4d05796ddb42cbb589e02fed55), there were some changes to the importer exit codes. So now, if there are duplicated transactions, the exit code is 1 (because there are errors), where before it was 0, resulting in false positives in my monitoring setup.

I'm not sure what would be a good solution, I would want to keep monitoring the daily import, and only be notified if something fails. Any ideas?

Thanks!!

**Comment by JC5** [comment, marked as answer]
Hey, sorry to hear that this breaks your import.

Try setting `IGNORE_DUPLICATE_ERRORS=true`, to make the data importer ignore duplication errors.

**Reply by danisabin** [reply]
Hi,
Oops, I missed that variable. It works as expected :)
Thank you so much for the reply and this awesome project!
